### ISSUE TITLE
License issue

### ISSUE THREAD

**Issue by grusell** [wader/static-ffmpeg#340]
Great project, very useful!
However, it seems the binaries built by this dockerfile are non-distributable, as far as I can tell this is caused by the inclusion of `libfk-aac` . I would suggest removing libdfk-aac to make the image distributable, at least for the image published on dockerhub.
Perhaps it might also be worth providing a dockerimage with just the dependencies (basically the builder stage without the ff* binaries) to make it easy to build a static ffmpeg dockerimage with libfdk-aac locally.

```$ docker run mwader/static-ffmpeg:6.0 -L
ffmpeg version 6.0 Copyright (c) 2000-2023 the FFmpeg developers
  built with gcc 12.2.1 (Alpine 12.2.1_git20220924-r4) 20220924
  configuration: --pkg-config-flags=--static --extra-cflags=-fopenmp --extra-ldflags='-fopenmp -Wl,-z,stack-size=2097152' --toolchain=hardened --disable-debug --disable-shared --disable-ffplay --enable-static --enable-gpl --enable-version3 --enable-nonfree --enable-fontconfig --enable-gray --enable-iconv --enable-libaom --enable-libaribb24 --enable-libass --enable-libbluray --enable-libdav1d --enable-libdavs2 --enable-libfdk-aac --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libkvazaar --enable-libmodplug --enable-libmp3lame --enable-libmysofa --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-librabbitmq --enable-librav1e --enable-librtmp --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libsrt --enable-libssh --enable-libsvtav1 --enable-libtheora --enable-libtwolame --enable-libuavs3d --enable-libvidstab --enable-libvmaf --enable-libvo-amrwbenc --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs2 --enable-libxml2 --enable-libxvid --enable-libzimg --enable-openssl
  libavutil      58.  2.100 / 58.  2.100
  libavcodec     60.  3.100 / 60.  3.100
  libavformat    60.  3.100 / 60.  3.100
  libavdevice    60.  1.100 / 60.  1.100
  libavfilter     9.  3.100 /  9.  3.100
  libswscale      7.  1.100 /  7.  1.100
  libswresample   4. 10.100 /  4. 10.100
  libpostproc    57.  1.100 / 57.  1.100
This version of ffmpeg has nonfree parts compiled in.
Therefore it is not legally redistributable.
```

**Comment by wader** [comment]
Hey and thanks! Yeah i've thought about doing something about but lacked energy :) to keep it simple i think i would lean towards keep it but commented out or possibly make it into an optional `--build-arg` somehow?

**Comment by grusell** [comment]
Sounds reasonable. I'll make a PR with a kind of build-arg-switch to enable libfdk-aac.

**Comment by wader** [comment]
Out of curiosity: do you plan on redistributing the binaries yourself somehow?

**Comment by grusell** [comment]
I'm looking to include the binaries in a docker image that I may want to make public.

**Comment by wader** [comment]
Fixed by #341

**Comment by wader** [comment]
FYI tagged 6.0-1 so there will soon be a 6.0-1 image tag
